scripts/provision.sh

```diff
@@ -0,0 +1,261 @@
+#!/bin/bash
+
+set -e # fail on error
+
+SCRIPT_DIR=$(dirname $0)
+
+# Read the platform configuration
+TARGET_PLATFORM=$1
+case $TARGET_PLATFORM in
+  aws)
+    STEMCELL=light-bosh-stemcell-3069-aws-xen-hvm-ubuntu-trusty-go_agent.tgz
+    ;;
+  gce)
+    STEMCELL=light-bosh-stemcell-2968-google-kvm-ubuntu-trusty-go_agent.tgz
+    STEMCELL_URL=http://storage.googleapis.com/bosh-stemcells/$STEMCELL
+    ;;
+  *)
+    echo "Must specify the target platform: gce|aws"
+    exit 1
+    ;;
+esac
+
+# Include functions to setup GCE BOSH networking
+. $SCRIPT_DIR/gce-assign-fixed-ip.sh
+
+# Include the terraform output variables
+. $SCRIPT_DIR/terraform-outputs-${TARGET_PLATFORM}.sh
+
+BOSH_ADMIN_USER=${BOSH_ADMIN_USER:-admin}
+BOSH_ADMIN_PASS=${BOSH_ADMIN_USER:-admin}
+BOSH_IP=${BOSH_IP:-$terraform_output_bosh_ip}
```

Comment:
This will cause issues on AWS, because the variable `bosh_ip` in `aws/outputs.tf` is set to the public IP instead of the private IP. 

We need to change `aws/outputs.tf` to use the private IP instead.